### PR TITLE
fvntr: changing question copy

### PR DESCRIPTION
**Pull request by fvntr**
clarifying this question due to some confusion

---

> ✅️ By submitting this PR, I have verified the following

- [x] Checked to [see if a similar PR has already been opened](https://github.com/techqueria/website/pulls) 🤔️
- [x] Reviewed the [contributing guidelines](https://github.com/techqueria/website/blob/master/.github/CONTRIBUTING.md) 🔍️
- [x] Added my name to the bottom of the list under the **Contributors** section in the [README.md](https://github.com/techqueria/website/blob/master/README.md) with a link to my personal website or GitHub profile 👥️ (Optional)

### CODE REVIEW

File: layouts/shortcodes/fields/latinx.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="field">
+  <label class="label">Do you identify as a Latinx / a person of Latin American descent?*</label>
+  {{ if .Get "help" }}
+  <p class="help">{{ .Get "help" }}</p>{{ end }}
+  <div class="control">
+    <label class="radio">
+      <input type="radio" aria-label="Do you identify as a Latinx?" name="identify_as_latinx_in_tech" value="Yes" required> Yes
```
Comment (fvcproductions) [inline layouts/shortcodes/fields/latinx.html]: name needs to be changed because that's what netlify uses for form integration with slack

Comment (fvcproductions) [reply layouts/shortcodes/fields/latinx.html]: so "identify_as_latinx" would suffice

File: layouts/shortcodes/fields/tech.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="field">
+  <label class="label">Are you actively in the tech industry?*</label>
```
Comment (fvcproductions) [inline layouts/shortcodes/fields/tech.html]: I would reword as "Are you currently a professional in the tech industry or interested in becoming one?"

File: layouts/shortcodes/fields/latinx.html (hunk 1 above)
Comment (fvntr) [reply layouts/shortcodes/fields/latinx.html]: updated

File: layouts/shortcodes/fields/tech.html (hunk 2 above)
Comment (fvntr) [reply layouts/shortcodes/fields/tech.html]: updated

File: content/english/communities/slack/join.md [hunk 3]
```diff
@@ -40,7 +40,8 @@ If you do not identify as Latinx, you will still receive an invitation as a mult
   {{< fields/member >}}
   {{< fields/linkedin >}}
   {{< fields/location >}}
-  {{< fields/latinx_in_tech help="You will get an invite either way." >}}
+  {{< fields/latinx help="You will get an invite either way." >}}
+   {{< fields/tech help="You will get an invite either way." >}}
```
Comment (fvcproductions) [inline content/english/communities/slack/join.md]: we can remove extra space in beginning and also remove help message for tech question

File: layouts/shortcodes/fields/tech.html (hunk 2 above)
Comment (fvcproductions) [reply layouts/shortcodes/fields/tech.html]: doesnt look like this was reworded

Comment (fvcproductions) [reply layouts/shortcodes/fields/tech.html]: the aria labels were but not label

File: layouts/shortcodes/fields/latinx.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<div class="field">
+  <label class="label">Do you identify as a Latinx / a person of Latin American descent?*</label>
```
Comment (fvcproductions) [inline layouts/shortcodes/fields/latinx.html]: i would remove a Latinx so it's just Latinx

do you identify as Latina

File: content/english/communities/slack/join.md (hunk 3 above)
Comment (fvntr) [reply content/english/communities/slack/join.md]: updated

File: layouts/shortcodes/fields/latinx.html (hunk 4 above)
Comment (fvntr) [reply layouts/shortcodes/fields/latinx.html]: updated

File: layouts/shortcodes/fields/tech.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
```
Comment (fvcproductions) [inline layouts/shortcodes/fields/tech.html]: label still needs to be updated

**Comment by fvcproductions** [comment]
mkk LGTM - note to self that Zapier integration will need to be updated
